### PR TITLE
GlobalGiving Initial Datasets

### PR DESCRIPTION
**Pull request by arynh**
Resolves issue #5 

* Part of bigger issue, #3

### CODE REVIEW

**Comment by JoshBurke** [comment]
TODO Osama and Lam:
-extract api_key into another file
-put try/except somewhere else
-remove unlabeled results list
-change time sleep value
-double quotes for docstrings
-Use dictionary.get()

**Comment by JoshBurke** [comment]
Also: vet for duplicate orgs

File: src/crawler.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    # loads global giving api key
+    load_dotenv()
+    global_giving_key = os.getenv("GLOBAL_GIVING_KEY")
+
+    # Specifying API to return JSON
+    headers = {"Accept": "application/json"}
+
+    # JSON files to write to
+    projects_json = open("projects.json", "w")
```
Comment (arynh) [inline src/crawler.py]: Make sure you close a file after you're done writing! A cleaner way to do this with Python is to say:
```python
with projects_json as open("projects.json", "w"):
    # write stuff
```

File: src/crawler.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        # Recording projects
+        projects_list += [
+            parse_project_info(project) for project in projects["project"]
+        ]
+        time.sleep(0.5)
+
+    # Removing duplicate organizations
+    projects_list = removeDuplicateOrganizations(projects_list)
+
+    # Writing projects to JSON file
```
Comment (arynh) [inline src/crawler.py]: This is the area in which you would need to use your `with` statement so the file is only open as long as you are writing in the scope of the `with` statement.

File: src/crawler.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+            # Recording orgs
+            orgs_list += [parse_org_info(org) for org in orgs["organization"]]
+            time.sleep(0.5)
+
+        # Writing orgs to JSON file
+        json.dump(orgs_list, orgs_json, sort_keys=True, indent=2, ensure_ascii=False)
+
+
+def get_org_key(org, keys):
```
Comment (JoshBurke) [inline src/crawler.py]: Is this method still necessary?

**Review by JoshBurke — COMMENTED** [review]
Looks good - except is the get_org_key method still necessary?

**Comment by osaesm** [comment]
It's not necessary; it's more of a cleanliness thing.

File: src/crawler.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            # Recording orgs
+            orgs_list += [parse_org_info(org) for org in orgs["organization"]]
+            time.sleep(0.5)
+
+        # Writing orgs to JSON file
+        json.dump(orgs_list, orgs_json, sort_keys=True, indent=2, ensure_ascii=False)
+
+
+def parse_org_info(org):
+    """
```
Comment (arynh) [inline src/crawler.py]: this is a really nice docstring

**Review by Lam7150 — APPROVED** [review]
Looks good!

**Pull request merged**